src/components/Sing/SequencerNote.vue

```diff
@@ -215,130 +264,267 @@ const onLeftEdgeMouseDown = (event: MouseEvent) => {
 </script>
 
 <style scoped lang="scss">
-@use "@/styles/variables" as vars;
-@use "@/styles/colors" as colors;
+@use "@/styles/v2/variables" as vars;
+@use "@/styles/v2/colors" as colors;
+
+@mixin text-outline($color, $size: 1px) {
+  text-shadow:
+    #{-$size} #{-$size} 0 $color,
+    #{$size} #{-$size} 0 $color,
+    #{-$size} #{$size} 0 $color,
+    #{$size} #{$size} 0 $color,
+    0 #{-$size} 0 $color,
+    0 #{$size} 0 $color,
+    #{-$size} 0 0 $color,
+    #{$size} 0 0 $color;
+}
 
 .note {
   position: absolute;
   top: 0;
   left: 0;
 
-  &.below-pitch {
-    .note-bar {
-      background-color: rgba(colors.$primary-rgb, 0.18);
-      border-color: hsl(130, 35%, 78%);
-    }
+  .note-bar {
+    box-sizing: border-box;
+    position: absolute;
+    width: calc(100% + 1px);
+    height: 100%;
+    background-color: var(--scheme-color-sing-note-bar-container);
+    border: 1px solid var(--scheme-color-sing-note-bar-border);
+    border-radius: 5px;
+  }
+
+  .note-left-edge,
+  .note-right-edge {
+    position: absolute;
+    top: 0;
+    width: 25%;
+    min-width: 3px;
+    max-width: 8px;
+    height: 100%;
+  }
+
+  .note-left-edge {
+    left: -2px;
+    border-radius: 5px 0 0 5px;
```

Comment:
こうやって見た目以上の幅を持たせるの、もしかしたらノートを隣に置いたときの挙動がちょっと微妙になるかもですね！
多分左側のノートの右端のエッジは当たり判定が2ピクセル小さくなってる気がします。

ただこれ解決するのちょっと難しいと思うので、問題があるということをFIXMEコメントで書いておくのとかどうでしょう 👀 
こんな感じ･･･？
```suggestion
  .note-left-edge {
    left: -2px; // FIXME: 左隣のノートの右端の当たり判定に食い込んでしまう
    border-radius: 5px 0 0 5px;
```